node.go

```diff
@@ -19,6 +19,7 @@ type Node interface {
 
 	AppendChild(n Node)
 	AttachShadow() Node
+	ClassList() *ClassList
```

Comment:
According to [docs](https://developer.mozilla.org/en-US/docs/Web/API/Element) this should be a function on `dom.Element`, not `dom.Node`.